### DISCUSSION TITLE
Unable to upload any CSVs

### DISCUSSION THREAD

**Discussion by Cody-Wise** [supabase/supabase#5953] [Questions]
I have tried to upload several different CSVs that are formatted with a header row and do not include special characters and I keep getting this error:

 https://www.dropbox.com/s/j2aehxwxgcd2x0r/2022-03-18_14-15-24.png?dl=0

I even exported a CSV from Supabase and tried to re-import and I get the same error.

Any suggestions?

**Comment by GaryAustin1** [comment]
I've not used csv upload, but tested it on Microsoft Windows 10 and the file type is "application/vnd.ms-excel" for a .csv file generated by Libre Office and notepad++ and the download from Supabase table editor.  
Supabase table upload is looking for file type text/csv so rejects it. 
FWIW

**Comment by hjaber** [comment]
`Sorry! We only accept CSV or TSV File types, please upload another file.`

I'm also unable to upload any .csv files currently and previously have uploaded multiple large .csv files without issue.

I have tried uploading an exported .csv file from excel, pgadmin, and google sheets with no luck. I have also created simple 2x2 .csv files in excel and google sheets without any luck.  Attempted on Windows 10 Pro Google Chrome and Brave Browser.

![image](https://user-images.githubusercontent.com/41503068/160675249-2cb60b61-4ba4-4a7d-af8c-023dbf1b2664.png)

**Comment by GaryAustin1** [comment]
The file type is this on Windows with "application/vnd.ms-excel".
There is an open issue https://github.com/supabase/supabase/issues/6012.
This appears to be a common issue: https://christianwood.net/posts/csv-file-upload-validation/

**Comment by nonniv** [comment]
Hi I ran into this, the solution for me was to have the column names in the header of the cvs in same case as the column name in the db.  To create the table with the column name's in camel case I needed to put quote around the column's name otherwise the default column name is lower case in the table
